Plugins/BuildNotifications.PluginInterfaces/Builds/Search/ISearchCriteria.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections.Generic;
+using JetBrains.Annotations;
+
+namespace BuildNotifications.PluginInterfaces.Builds.Search
+{
+    /// <summary>
+    /// Defines a criteria which may be used within the global search.
+    /// </summary>
+    [PublicAPI]
+    public interface ISearchCriteria
+    {
+        /// <summary>
+        /// The keyword to search by this criteria. Case-insensitive. Is expected to be localized.
+        /// </summary>
+        /// <example>
+        /// For a search criteria by which user the keyword could be 'RequestedBy'. E.g. 'RequestedBy: Me'
+        /// </example>
+        string LocalizedKeyword { get; }
+
+        /// <summary>
+        /// A short, localized description of what this search criteria does.
+        /// </summary>
+        /// <example>
+        /// 'Specifies build of certain users.'
+        /// </example>
+        string LocalizedDescription { get; }
```

Comment:
A plugin can't localize values because it can't know which culture to use. The settings refactoring introduces an ``ILocalizer`` but we would still need a layer between a plugin and the actual translation.

Reply:
Soo use a method instead, which will receive the current culture?

Reply:
Sounds good